src/components/chip-set/chip-set.tsx

```diff
@@ -789,4 +804,20 @@ export class ChipSet {
 
         return <limel-badge label={chip.badge} />;
     }
+
+    private triggerIconColorWarning(value: Chip[]) {
+        for (const chip of value) {
+            if (
+                chip.icon &&
+                (chip.iconFillColor ||
+                    chip.iconBackgroundColor ||
+                    chip.iconTitle)
+            ) {
+                /* eslint-disable-next-line no-console */
+                console.warn(
```

Comment:
it is enough that we have the `console.warn` in `limel-chip-set` only. Bacuse `limel-picker` and `limel-file` are using `limel-chip-set`, and will show the console warning

Reply:
I think we need to reconsider where to place the console.warn because right now it feels a bit aggressive and the console just fills with the warnings. 😅

Reply:
It's also not very easy to read/understand because of the lack of formatting in the console messages. 
Maybe we could create a short documentation that explains the difference and link there from the console.warns instead?

Reply:
It's not terrible, but this is only after clicking a couple of times in the browser. 
<img width="504" alt="Screenshot 2023-11-09 at 11 35 02" src="https://github.com/Lundalogik/lime-elements/assets/122509723/d7e1063b-be3e-4f0c-8b78-917123e4b54e">

Reply:
Well, I'd say this is actually the least aggressive way of doing it 😄 and the experience shows that this is not enough for consultants and appers to update their components. After this PR is merged, we should update our code in crm-components and building-blocks, and use the new `Icon` interface ourselves. Then we won't see these errors anymore. :)

Reply:
Do we already have issues created for those?

Reply:
no. But I won't forget this :D